protocol/protocol.tex

```diff
@@ -1026,7 +1033,8 @@
 
 A \keyTuple $(\AuthPrivate, \TransmitPrivate, \PaymentAddress)$ is
```

Comment:
We mentioned key tuples in only two places, so I've simplified the spec (in my local copy) by removing that concept and just referring to the key components.